### PR TITLE
New and updated FAQ's

### PR DESCRIPTION
**Pull request by johnswil**
Hi, thank you for your contribution to Mantl! Before we can accept any code into
master, we need it to meet the following criteria. If there are any you can't
satisfy yourself, go ahead and open the pull request anyway and we'll help you
test. Feel free to delete this message once you're done. Thanks again!

- [ ] Installs cleanly on a fresh build of most recent master branch
- [ ] Upgrades cleanly from the most recent release
- [x ] Updates documentation relevant to the changes

### CODE REVIEW

File: docs/faq.rst [hunk 1]
```diff
@@ -1,11 +1,32 @@
 FAQs
 ====
 
-How does Mantl compare to `OpenStack Magnum <https://wiki.openstack.org/wiki/Magnum>`_?
+How are Mantl components chosen_?
+--------------------------------------------------------------------------------------
+
+A service design methodology was used to assemble Mantl components. Priority was given
+to sensing the large shifts in this technology and to defining an architecture 
+consistent with mode 1 to mode 2+ methodologies. Once the architecture was optimized
```
Comment (siddharthist) [inline docs/faq.rst]: What are these "modes" you're talking about?

File: docs/faq.rst [hunk 2]
```diff
@@ -1,11 +1,32 @@
...
 ====
 
-How does Mantl compare to `OpenStack Magnum <https://wiki.openstack.org/wiki/Magnum>`_?
+How are Mantl components chosen_?
+--------------------------------------------------------------------------------------
+
+A service design methodology was used to assemble Mantl components. Priority was given
+to sensing the large shifts in this technology and to defining an architecture 
+consistent with mode 1 to mode 2+ methodologies. Once the architecture was optimized 
+feedback was requested from the enterprise developers and SP partners. Popular open source
```
Comment (siddharthist) [inline docs/faq.rst]: What is an SP partner?

File: docs/faq.rst [hunk 3]
```diff
@@ -16,22 +37,31 @@ infrastructure. Mantl hosts are provisioned to OpenStack and other supported
 environments using `Terraform <https://www.terraform.io/>`_. Terraform
 configuration files manage OpenStack services such as compute,
 block storage, networking, etc. required to instantiate a Mantl host
-to an OpenStack cloud. The Terraform `OpenStack Provider`_ would need to be
+to an OpenStack cloud. The Terraform `OpenStack Provider
+<https://www.terraform.io/docs/providers/openstack/index.html>`_ would need to be
 updated since it does not support Magnum. If/when this is accomplished, adding
-Magnum support to Mantl should be simple.
+Magnum support to Mantl should be straight forward.
```
Comment (siddharthist) [inline docs/faq.rst]: "straightforward"

**Comment by siddharthist** [comment]
@johnswil I left you some comments. Additionally, I think that the first question/answer pair you've added is pretty abstract (and long) and might belong in a new document (e.g. "Design Philosophy"). It doesn't really fit in with the other questions in the FAQ. Thanks for the contribution!

File: docs/faq.rst (hunk 2 above)
Comment (johnswil) [inline docs/faq.rst]: Service provider partner

File: docs/faq.rst (hunk 1 above)
Comment (johnswil) [inline docs/faq.rst]: Mode 1 is traditional and sequential, emphasizing safety and accuracy. Mode 2 is exploratory and nonlinear, emphasizing agility and speed.

File: docs/faq.rst (hunk 2 above)
Comment (siddharthist) [inline docs/faq.rst]: That should be clarified here, I am one of the main committers to Mantl and don't know what it is :smile:

File: docs/faq.rst (hunk 3 above)
Comment (johnswil) [inline docs/faq.rst]: Yes, one word ;-)

File: docs/faq.rst (hunk 1 above)
Comment (siddharthist) [inline docs/faq.rst]: We shouldn't expect that anyone will know that terminology, I've personally never heard it.

Comment (johnswil) [inline docs/faq.rst]: Agreed, let's change as follows...
line 9 : consistent with traditional as well as Agile methodologies.....

File: docs/faq.rst (hunk 2 above)
Comment (johnswil) [inline docs/faq.rst]: Agreed, should just read "service providers"  no longer "SP partners".

File: docs/faq.rst (hunk 3 above)
Comment (johnswil) [inline docs/faq.rst]: These two changes should now be merged, please advise

**Comment by siddharthist** [comment]
@johnswil Can you please make a new document ("Design Philosophy") for your first Q/A pair?

**Comment by johnswil** [comment]
This is ready to be merged, design philosophy has been removed.

File: docs/faq.rst [hunk 4]
```diff
@@ -16,22 +16,31 @@ infrastructure. Mantl hosts are provisioned to OpenStack and other supported
...
 
-Mantl is an end-to-end solution for deploying and managing a microservices
-infrastructure. Mantl currently uses `Apache Mesos <http://mesos.apache.org/>`_
-as a cluster manager for microservices. The Mantl team is in the process of
-evaluating Kubernetes as a cluster manager.
+Mantl has integrated both Apach Mesos and Kubernetes into it's container stack.
+This integration provides users the freedom to choose the best scheduler for their
+workloads promoting greater flexibility and choice.
+
+Containers are great for running stateless applications but what about data/stateful services_?
```
Comment (siddharthist) [inline docs/faq.rst]: What's the underscore here for?

**Comment by siddharthist** [comment]
@johnswil I've left just one more comment. Sorry I didn't catch it the first time around!

**Comment by johnswil** [comment]
Who put that there! lol, no that underscore does not belong there ;-)

**Comment by johnswil** [comment]
Ready for merge

File: docs/faq.rst [hunk 5]
```diff
@@ -16,22 +16,31 @@ infrastructure. Mantl hosts are provisioned to OpenStack and other supported
...
 workloads to ensure that their state matches the users declared intentions.
 Using the concepts of "labels" and "pods", it groups the containers which
-make up an application into logical units for easy management and discovery.
+make up an application into logical units for management and discovery.
 
-Mantl is an end-to-end solution for deploying and managing a microservices
-infrastructure. Mantl currently uses `Apache Mesos <http://mesos.apache.org/>`_
-as a cluster manager for microservices. The Mantl team is in the process of
-evaluating Kubernetes as a cluster manager.
+Mantl has integrated both Apach Mesos and Kubernetes into it's container stack.
```
Comment (siddharthist) [inline docs/faq.rst]: This should read "Apache Mesos"

**Comment by siddharthist** [comment]
@johnswil There's one more typo in there.

**Comment by johnswil** [comment]
We don't want Apache Mesos in quotes, siddharthist why is this necessary?

**Comment by siddharthist** [comment]
@johnswil You're right, we don't need to quotes. Ignore the quotes and read again. You have a typo ("Apach Mesos") that needs to be changed to "Apache Mesos".

**Pull request merged**